main.js

```diff
@@ -0,0 +1,255 @@
+                key.classList.add('active');
+            }
+        });
+
+        if (event.ctrlKey && event.altKey) {
+            allKeys.forEach((key) => {
+                if (!key.dataset.keyProp.includes('Digit') && !key.classList.contains('spec-key')) {
+                    key.querySelectorAll('span').forEach((span) => {
+                        span.classList.toggle('hidden');
+                    })
+                }
+            })
+        }
+
+        printLetter();
+
+
+
+        output.focus();
+        output.selectionStart = output.value.length;
+
+
+    })
+
+    document.addEventListener('keyup', function (event) {
+        const allKeys = [...document.querySelectorAll('.key')];
+
+        if (event.code === 'ShiftLeft' || event.code === 'ShiftRight') {
+            allKeys.forEach((key) => {
+                if (!key.classList.contains('spec-key')) {
+                    key.querySelectorAll('span').forEach((span) => {
+                        if (!span.classList.contains('hidden')) {
+                            span.innerHTML = span.innerHTML.toLowerCase();
+                        }
+                    })
+                }
+            })
+        }
+
+        allKeys.forEach((key) => {
+            if (event.code === key.dataset.keyProp) {
+                key.classList.remove('active');
+            }
+        })
+
+
+    })
+
+    function printLetter() {
+        const allKeys = document.querySelectorAll('.key');
+
+
+        allKeys.forEach((key) => {
+            if (key.classList.contains('active') && key.classList.contains('enter')) {
+                output.value += '\n';
+            } else if (key.classList.contains('active') && key.classList.contains('tab')) {
+                output.value += '    ';
+            } else if (key.classList.contains('active') && key.classList.contains('backspace')) {
+                let pos =
+                    output.value = output.value.slice(0, -1);
+            } else if (key.classList.contains('space') && key.classList.contains('active')) {
+                output.value += ' ';
+            } else if (key.classList.contains('active') && !key.classList.contains('spec-key')) {
+                key.querySelectorAll('span').forEach((span) => {
+                    if (!span.classList.contains('hidden')) {
+                        output.value += span.innerText;
+                    }
+                })
+            }
+        })
+
+    }
+
+
+    keyboard.addEventListener('mousedown', function (event) {
+
+
+        if (event.target.classList.contains('key')) {
+            event.target.classList.add('active');
+        }
+        printLetter();
+        output.focus();
+        output.selectionStart = output.value.length;
+    })
+
+    keyboard.addEventListener('mouseup', function (event) {
+
+        if (event.target.classList.contains('key')) {
+            event.target.classList.remove('active');
+        }
+
+
+    })
+
```

Comment:
Please remove comments from the code and correct the alignment. No more than one line between code blocks